toolkit/authentication/src/androidTest/java/com/arcgismaps/toolkit/authentication/UiDeviceExtensions.kt

```diff
@@ -0,0 +1,80 @@
+package com.arcgismaps.toolkit.authentication
+
+import android.app.Activity
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until
+import org.junit.Assert
+
+
+/**
+ * Waits for the View with the matching package to be visible. Throws an error if the view can't be
+ * found.
+ *
+ * @param packageId the view's package Id to wait for.
+ * @since 200.4.0
```

Comment:
Needs updating also in other places in this file...
```suggestion
 * @since 200.5.0
```